packages/ffe-form-react/src/index.d.ts

```diff
@@ -25,6 +25,7 @@ export interface BaseFieldMessageProps
     extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Here HTMLDivElement should be changed to HTMLElement as it is not known anymore that this is a div-element.